README.md

```diff
@@ -63,16 +63,41 @@ const thx = msgFV["no"];
 
 **Cluster**
 
+> This is an experimental feature and may be subject to breaking changes even after a
+major release see [here](experimental/README.md)
+
+First, if you need to set up nodes into a working redis cluster:
 ```ts
 import { connect } from "https://deno.land/x/redis/mod.ts";
 
-const redis = await connect({ hostname: "127.0.0.1" });
+const redis = await connect({ hostname: "127.0.0.1", port: 6379 });
+
+// connect each node to form a cluster (see https://redis.io/commands/cluster-meet)
 await redis.clusterMeet("127.0.0.1", 6380);
+...
+
+// List the nodes in the cluster
 await redis.clusterNodes();
 // ... 127.0.0.1:6379@16379 myself,master - 0 1593978765000 0 connected
 // ... 127.0.0.1:6380@16380 master - 0 1593978766503 1 connected
 ```
 
+To consume a redis cluster:
```

Comment:
The documentation for the Redis Cluster client is already available in `experimental/cluster/README.md`. Could you put a link to it instead of example code?
```suggestion
To consume a redis cluster, you can use [experimental/cluster module](experimental/cluster/README.md).
```